pages/positions/openloan.tsx

```diff
@@ -1,10 +1,295 @@
-import type { NextPage } from 'next'
-import OpenLoanController from '../../components/OpenLoan/OpenLoanController'
+import type {NextPage} from 'next'
+import {useState, useEffect, Dispatch, SetStateAction, useCallback, ChangeEvent} from 'react'
+import {Token} from '../../components/Tokens'
+import {CollateralType} from '../../components/OpenLoan/CollateralType'
+import SelectCollateralModal from '../../components/OpenLoan/SelectCollateralModal'
+import {FieldValues, useForm} from 'react-hook-form'
+import useNotification from '../../hooks/useNotification'
+import {FaInfoCircle} from 'react-icons/fa'
+import {ChevronDownIcon} from '@heroicons/react/solid'
+import PairsSelector from '../../components/PairsSelector'
+import {useOpenLoanHandler} from '../../hooks/useOpenLoanHandler'
+
+const style = {
+  wrapper: 'w-screen flex justify-center items-center',
+  formContent: 'bg-gray-900 w-[30rem] rounded-2xl p-4 max-w-lg',
+  vStack: 'items-center flex-col',
+  vStackItem: 'mt-3',
+  formHeader: 'justify-between items-center font-semibold text-xl text-gray-200 text-center',
+  numberInputContainer: 'bg-gray-800 rounded-2xl p-4 border-2 border-gray-800 hover:border-gray-600 flex justify-between w-full',
```

Comment:
if the styling starts getting this big, it's best to separate it to its own file

Reply:
Done